demo/data.js

```diff
@@ -179,7 +179,7 @@ export const demoData = {
       rows: {},
       cols: {},
       merges: [],
-      cells: { B2: { content: "42" } },
+      cells: { B2: { content: "42" }, D2: { content: "Bar" } },
```

Comment:
If you encounter a situation shere you have to update the raw data in place, it means you need a migration for the said data.

Reply:
I didn't fully understand your point. Are you referring to the need to write a migration in `migration_step.ts`? Could you please elaborate further?